style.css

```diff
@@ -276,18 +276,115 @@ header {
   background-color: #dfe1e6;
 }
 
+/* contact form */
+#contactForm-wrapper {
+  background-color: #fff;
+}
+
+#contact-form {
+  display: flex;
+  flex-direction: column;
+  height: 779px;
+  background-color: #6070ff;
+  background-image: url('/images/contact-form.png');
+  background-size: 265px 709px;
+  background-repeat: no-repeat;
+  background-position: right;
+  background-origin: content-box;
+  background-position-y: 21px;
+  border-top-left-radius: 100px;
+  text-align: center;
+}
+
+#contact-form h1 {
+  padding-top: 103px;
+  margin-bottom: 12px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+}
+
+#contact-form p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 140%;
+  color: #ebebff;
+  margin-left: 24px;
+  margin-right: 24px;
+  text-align: center;
+  margin-bottom: 87px;
+}
+
+#contact-form input[type=text],
+#contact-form input[type=email] {
+  height: 48px;
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin-bottom: 20px;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 118%;
+  color: #172b4d;
+  padding-left: 16px;
+}
+
+#contact-form form {
+  display: flex;
+  flex-direction: column;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+#contact-form textarea {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 160%;
+  color: #b3bac5;
+  height: 140px;
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding-top: 15px;
+  padding-left: 16px;
+}
+
+#contact-form button[type=submit] {
+  width: 142px;
+  height: 48px;
+  background: #fff;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 141%;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  border: none;
+  margin-top: 22px;
+}
+
```

Comment:
- [ ] **[Optional]** I think it would be better to add the button interaction such as hover, active (when it's pressed), disabled, and enable (default state)  styled as shown in the Figma template. 

You may kindly follow this [link ](https://www.educative.io/answers/how-to-create-an-animated-button-using-html-and-css)for more info 😎

![image](https://user-images.githubusercontent.com/102757126/214645289-391e6ee5-3533-4dee-89df-26e0f3e439f4.png)